ocaml/runtime/domain.c

```diff
@@ -829,15 +774,9 @@ static void domain_create(uintnat initial_minor_heap_wsize,
   domain_state->trap_barrier_block = -1;
 #endif
 
-<<<<<<< HEAD
-  add_to_stw_domains(domain_self);
-||||||| 121bedcfd2
-  caml_reset_young_limit(domain_state);
-  add_to_stw_domains(domain_self);
-=======
-  caml_reset_young_limit(domain_state);
+  // XXX mshinwell: is this line needed?  It was deleted on our side
+  // caml_reset_young_limit(domain_state);
```

Comment:
It should remain deleted.

It was removed upstream as part of the statmemprof work (12923), and backported here as part of #2461. (These changes didn't make it into 5.2)